prowler

```diff
@@ -320,6 +325,24 @@ if [[ -n "$ALLOWLIST_FILE" ]]; then
   allowlist
 fi
 
+# Check database providers
+if [[ ${DATABASE_PROVIDER} ]]
+then
+    # Check if input provider is supported
+    if ! grep -q "${DATABASE_PROVIDER}" <<< "${SUPPORTED_DB_PROVIDERS}"
```

Comment:
I think `-E` is missing here. 

If `DATABASE_PROVIDER="postgresql|mariadb"` you need to specify this option to grep it.

Reply:
Completely